### PR TITLE
[WIP] NODE-956 DOCS Examples

### PR DESCRIPTION
**Pull request by christkv**

### CODE REVIEW

File: test/functional/examples_tests.js [hunk 1]
```diff
@@ -0,0 +1,969 @@
...
+            tags: ["blank", "red"],
+            size: { h: 14, w: 21, uom: "cm" }},
+          { item: "mat",
+            qty: 85,
+            tags: ["gray"],
+            size: { h: 27.9, w: 35.5, uom: "cm" }},
+          { item: "mousepad",
+            qty: 25,
+            tags: ["gel", "blue"],
+            size: { h: 19, w: 22.85, uom: "cm" }}])
```
Comment (mbroadst) [inline test/functional/examples_tests.js]: nitpick: close bracket on next line

File: test/functional/examples_tests.js [hunk 2]
```diff
@@ -0,0 +1,969 @@
...
+            qty: 100,
+            size: { h: 8.5, w: 11, uom: "in" },
+            status: "D"},
+          { item: "planner",
+            qty: 75, size: { h: 22.85, w: 30, uom: "cm" },
+            status: "D"},
+          { item: "postcard",
+            qty: 45,
+            size: { h: 10, w: 15.25, uom: "cm" },
+            status: "A"}])
```
Comment (mbroadst) [inline test/functional/examples_tests.js]: nit: close bracket on next line

File: test/functional/examples_tests.js [hunk 3]
```diff
@@ -0,0 +1,969 @@
...
+        // Start Example 10
+        var cursor = db.collection('inventory').find({ 
+          status: { $in: ["A", "D"] }
+        });
+        // End Example 10
+        assert.equal(5, yield cursor.count());
+
+        // Start Example 11
+        var cursor = db.collection('inventory').find({ 
+          status: "A", qty: { $lt: 30 }
```
Comment (mbroadst) [inline test/functional/examples_tests.js]: nit: keys on same line

**Comment by christkv** [comment]
did fixes and merged

**Pull request merged**